about.html

```diff
@@ -0,0 +1,143 @@
+                    with your schedule. All it takes is to register your complimentary Alison account and thousands of
+                    free courses will be at your fingertips.</p>
+            </div>
+            <div class="col-10">
+
+                <img src="images/logo.png" alt="logo" class="mx-auto d-block p-lg-5 p-md-5 p-3 border">
+            </div>
+        </div>
+
+
+    </section>
+    <section class="mb-5">
+        <div class="row justify-content-center ">
+            <div class="col-11 d-flex flex-column align-items-center mt-5">
+                <h2 class="text-center ">Our Past Highlights</h2>
+                <div class="line "></div>
+                <p class="mt-2 text-center col-8">We have been trying to improve day by day and including as much courses and quality content as possible</p>
+            </div>
+        </div>
+        <div class="row justify-content-center mx-5  ">
+            <div class="col-lg-4 col-11 col-md-5 p-lg-5 p-4 text-center m-2 background1 text-light">
+            <h2>2018</h2>
+            <p class="m-0">500+ Courses</p>
+            </div>
+            <div class="col-lg-4  col-11 col-md-5 p-lg-5 p-4 text-center  m-2 background2 text-light">
+
+                <h2>2020</h2>
+                <p class="m-0">1000+ courses</p>
+            </div>
+        </div>
+
+
+    </section>
+    <section class="bg-dark py-5 partner">
+        <div class="row justify-content-center ">
+            <div class="col-5 d-flex flex-column align-items-center ">
+                <h2 class="text-center text-secondary">Partner</h2>
+                <div class="line mb-4 "></div>
+            </div>
+        </div>
+        <div class="row justify-content-center">
+            <div class="col-lg-1 col-md-2 col-3 m-lg-3"><img src="images/partner/partner-1.png" alt="partner-1"></div>
+            <div class="col-lg-1 col-md-2 col-3 m-lg-3"><img src="images/partner/partner-2.png" alt="partner-2"></div>
+            <div class="col-lg-1 col-md-2 col-3 m-lg-3 "><img src="images/partner/partner-3.png" alt="partner-3"></div>
+
+            <div class="col-lg-1  col-md-2 col-4 m-lg-3"><img src="images/partner/partner-4.png" alt="partner-4"></div>
+            <div class="col-lg-1  col-md-2 col-3 m-lg-3"><img src="images/partner/partner-5.png" alt="partner-5"></div>
+        </div>
+    </section>
+    <footer class="py-5 ">
+        <div class="row justify-content-center ">
+            <div class="col-lg-3 col-md-4 col-6 align-self-center ">
+                <img src="images/logo.png" class="img-fluid" alt="logo">
+            </div>
+            <div class="col-5  align-self-center">
+                <p class="small">With more than 5,800 courses, Professional Certificates, and degrees from top-tier
+                    colleges and organisations, you may launch, change, or enhance your profession.</p>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please give the correct `BG` color to the `ABOUT` page's footer part and remove the `partners` part for this page 🙏🏼 

![image](https://github.com/batoolfatima2135/Capstone/assets/63915024/bb16ac1f-18b8-4fb9-8a0b-825a9723c943)